assets/timber.scss.liquid

```diff
@@ -1685,32 +1740,26 @@ label.error {
 .site-header {
   padding: ($gutter / 2) 0;
 
-  @include at-query ($min, $large) {
-    padding: $gutter 0;
-  }
+  .grid {
```

Comment:
this seems like a pretty big change for `.grid`, if it affects all grids inside of `.site-header`, what about creating a variant class for the grid in this case? Reading the markup, if there aren't any other clues that the grid changes, I'd be surprised by this behavior. Why is it necessary within the site-header?